internal/namespaces/instance/v1/custom_server.go

```diff
@@ -550,12 +551,20 @@ func serverDeleteCommand() *core.Command {
 				Required: true,
 			},
 			{
-				Name:  "delete-ip",
-				Short: "Delete the IP attached to the server as well",
+				Name:    "with-volumes",
+				Short:   "Delete the volumes attached to the server as well",
+				Default: core.DefaultValueSetter("none"),
+				EnumValues: []string{
+					"none",
+					"local",
+					"block",
+					"root_only",
+					"all",
+				},
 			},
 			{
-				Name:  "delete-volumes",
-				Short: "Delete the volumes attached to the server as well",
+				Name:  "with-ip",
+				Short: "Delete the IP attached to the server as well",
```

Comment:
`Deletes the IP in addition to the server it is attached to`

Reply:
`Deletes` ==> `Delete` (infinitive form is valid for manual)